runestone/khanex/js/exercises/absolute_value.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
Are all of these exercises needed as part of the RunestoneComponents?  It seems like this would be better to have as part of a book...

Also, I can't find anywhere in the code that these files are referenced.